web/unit_tests/test.sh

```diff
@@ -105,10 +115,22 @@ if [ $CONFIG = CI.conf.js ]; then
 fi
 
 BASE_PATH=`dirname $BASH_SOURCE`
+echo "$BASE_PATH"
```

Comment:
Delete this line? Or else 
```suggestion
echo "Base path: $BASE_PATH"
```